.github/workflows/node.js.yml

```diff
@@ -16,7 +20,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x, 14.x, 15.x]
+        node-version: [14.x, 15.x, 16.x, 18.x]
```

Comment:
Why are we stopping the tests for node 10 and 12? I know that [14 is the oldest one still maintained](https://nodejs.org/en/about/releases/) but we might have customers using it with Node 10/12, what do you think?

Reply:
Well looking at [here](https://nodejs.org/en/about/releases/), older supported node version is 14 and I believe version before are now unsupported.